testing/scenario_app/README.md

```diff
@@ -0,0 +1,42 @@
+# Scenario App
+
+This folder contains a dart:ui application and scripts to compile it to AOT
+for exercising embedders.
+
+It intentionally has no dependencies on the Flutter framework or tooling, such
+that it should be buildable as a presubmit or postsubmit to the engine even in
+the face of changes to Dart or dart:ui that require upstream changes in the
+Flutter tooling.
+
+To add a new scenario, create a new subclass of `Scenario` and add it to the
+map in `main.dart`. For an example, see animated_color_square.dart, which draws
+a continuously animating colored square that bounces off the sides of the
+viewport.
+
+## Building for iOS
+
+In this folder, after building the `ios_host` and `ios_profile` engine targets,
+run:
+
+```bash
+./compile_ios_aot.sh ../../../out/host_profile ../../../out/ios_profile/clang_x64/
+```
+
+This will create an `App.framework` copy it as well as the correct
+`Flutter.framework` to where the Xcode project expects to find them.
+
+You can then use `xcodebuild` to build the `ios/Scenarios/Scenarios.xcodeproj`,
+or open that in Xcode and build it that way.
+
+Currently, only AOT modes are supported.
+
+## Building for Android
+
+In this folder, after building the `android_host` and `android_profile_arm64`
```

Comment:
Is there a reason that we need to build for `profile` mode? Whenever I test, I like to run in `debug` mode so that I get the debug output to root cause failures. If we have the choice to use `debug` instead, can you add instructions here for that path?

Reply:
Right now, the scripts are only wired up to build AOT apps.  

Part of my goal here is to eventually have performance benchmarks based on engine commits, which we'd want to do AOT.

We could wire up JIT as a separt effort.

Reply:
So at this point, what would the expectations be on a developer writing a test that's failing, assuming that the stack trace alone is insufficient to root cause the problem?

Reply:
Right now that would be tough.  We can add a jit path too though.